test/e2e/kubernetes/service/service.go

```diff
@@ -58,10 +58,10 @@ type LoadBalancer struct {
 // Get returns the service definition specified in a given namespace
 func Get(name, namespace string) (*Service, error) {
 	cmd := exec.Command("kubectl", "get", "svc", "-o", "json", "-n", namespace, name)
-	util.PrintCommand(cmd)
```

Comment:
We don't need to output a bunch of `kubectl get svc` statements in our E2E stdout